Scenarioo/Scenarioo/Model/Docu/Entities/ScreenAnnotation.cs

```diff
@@ -0,0 +1,95 @@
+ * permission to link this library with independent modules, according 
+ * to the GNU General Public License with "Classpath" exception as provided
+ * in the LICENSE file that accompanied this code.
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+using System;
+using System.Xml.Serialization;
+
+using Scenarioo.Model.Docu.Entities.Generic;
+
+namespace Scenarioo.Model.Docu.Entities
+{
+    [Serializable]
+    [XmlRoot("screenAnnotation")]
+    public class ScreenAnnotation
+    {
+        [XmlElement("region")]
+        public ScreenRegion Region { get; set; }
+
+        /// <summary>
+        /// (optional) Set the visual style of the annotation (if not set, the same style as 'neutral' will be used).
+        /// </summary>
+        [XmlElement("style")]
+        public ScreenAnnotationStyle Style { get; set; }
+
+        /// <summary>
+        /// (optional) Set a separate title text, that is only displayed inside the popup window (and not on the screen).
+        /// This text is displayed instead of 'screenText' inside the popup.
+        /// </summary>
+        [XmlElement("title")]
+        public string Title { get; set; }
+
+        /// <summary>
+        /// (optional) Set the text to display inside the screen on the annotation (should be short, use description 
+        /// for longer texts). Too long texts might be truncated inside the screenshot view in the documentation.
+        /// </summary>
+        [XmlElement("screenText")]
+        public string ScreenText { get; set; }
+
+        /// <summary>
+        /// (optional) Set a longer textual description for this annotation, this annotation is displayed below 
+        /// the shorter 'text' inside an info popup that can be opened on the annotation.
+        /// </summary>
+        [XmlElement("description")]
+        public string Description { get; set; }
+
+        /// <summary>
+        /// (optional) Set a click action to define what happens, when the annotation is clicked on.
+        /// </summary>
+        [XmlElement("clickAction")]
+        public ScreenAnnotationClickAction? ClickAction { get; set; }
+
+        /// <summary>
+        /// (optional, but mandatory in case that clickAction is {@link ScreenAnnotationClickAction#toUrl})
+        /// The URL to browse to when the annotation is clicked on. The URL will be opened in a separate browser tab.
+        /// </summary>
+        [XmlElement("clickActionUrl")]
+        public string ClickActionUrl { get; set; }
+
+        [XmlElement("details")]
+        public Details Details { get; set; }
```

Comment:
@tobiaszuercher is it convenient for the client of the library, that details objects are not initialized as empty, such that he can simply add propoerties, if he likes to? I think in the Java library we initialize such objects as empty, for he can simply add properties on it.